Formula/molten-vk.rb

```diff
@@ -1,15 +1,12 @@
 class MoltenVk < Formula
   desc "Implementation of the Vulkan graphics and compute API on top of Metal"
   homepage "https://github.com/KhronosGroup/MoltenVK"
-  url "https://github.com/KhronosGroup/MoltenVK/archive/v1.0.41.tar.gz"
-  sha256 "a11208f3bc2eb5cd6cfebbc0bc09ac2af8ecbe5cc3e57cde817bc8bc96d2cc33"
+  url "https://github.com/KhronosGroup/MoltenVK/archive/v1.1.0.tar.gz"
+  sha256 "0538fa1c23ddae495c7f82ccd0db90790a90b7017a258ca7575fbae8021f3058"
   license "Apache-2.0"
-  revision 2
 
   bottle do
     cellar :any
-    sha256 "3eda5e70c4f70cdea90c46180574d6b61c8849ca2eeec033ab9c33bb3cae5280" => :catalina
-    sha256 "a6875344cb7327b6dd804a82ebd348fda81bb93d2160d0b756a2f133a38e7299" => :mojave
```

Comment:
please don't edit the bottle block

Reply:
Hey, the sha256 is outdated and the BigSur bottle is missing!
How can I fix that?

Reply:
By making a pull request and letting CI successfully build it. It'll then be updated on merge.